api/api/specs/routers.yaml

```diff
@@ -124,7 +133,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/RouterDetails"
+                $ref: "#/components/schemas/RouterVersion"
```

Comment:
This makes me uncomfortable. The operation is on `router` entity, so the response is expected to be the `RouterDetails` and not `RouterVersion`. 

The `RouterVersion` response would be expected, if the operation is done on `router_versions` (i.e. `POST /projects/{project_id}/routers/{router_id}/versions`)